main.js

```diff
@@ -1056,7 +1135,7 @@ class Tado extends utils.Adapter {
 		}
 		if (method != 'get') {
 			this.apiCallinExecution = true;
-			console.log(`Body "${JSON.stringify(data)}" for API call "${url}"`);
+			this.log.debug(`Body "${JSON.stringify(data)}" for API call "${url}"`);
```

Comment:
'console.log' was on purpose, this helps a lot for debugging sentry info. There the console log is available.
Logging both is an option.

Reply:
Makes sense. I used the iobroker logs to debug everything, therefore I changed it